extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -121,19 +120,19 @@ The attribute's accessor `type` must be `"SCALAR"` and `normalized` must be fals
 
 In some cases it is possible to define feature IDs implicitly, such as when all vertices in a primitive have the same feature ID or when each vertex in a primitive has a different feature ID.
 
-This is accomplished by using `constant` and `divisor`.
+This is accomplished by using `offset` and `repeat`.
 
-* `constant` sets a constant feature ID for each vertex. The default is `0`.
-* `divisor` sets the rate at which feature IDs increment. If `divisor` is zero then `constant` is used. If `divisor` is greater than zero the feature ID increments once per `divisor` sets of vertices, starting at `constant`. The default is `0`.
+* `offset` specifies the initial value for the vertex feature ID range. The default is `0`.
+* `repeat`, if defined, specifies the number of vertices for which to repeat each feature ID before incrementing the ID by 1. If `repeat` is undefined, the feature ID for all vertices is `offset`.
 
 For example
 
-* If `constant` is 0 and `divisor` is 0, the feature IDs are `[0, 0, 0, ...]`
-* If `constant` is 0 and `divisor` is 1, the feature IDs are `[0, 1, 2, ...]`
-* If `constant` is 0 and `divisor` is 2, the feature IDs are `[0, 0, 1, 1, 2, 2, ...]`
-* If `constant` is 2 and `divisor` is 3, the feature IDs are `[2, 2, 2, 3, 3, 3, 4, 4, 4, ...]`.
+* If `offset` is 0 or undefined and `repeat` is undefined, the feature IDs are `[0, 0, 0, ...]`
+* If `offset` is 0 and `repeat` is 1, the feature IDs are `[0, 1, 2, ...]`
+* If `offset` is 0 and `repeat` is 2, the feature IDs are `[0, 0, 1, 1, 2, 2, ...]`
+* If `offset` is 2 and `repeat` is 3, the feature IDs are `[2, 2, 2, 3, 3, 3, 4, 4, 4, ...]`.
```

Comment:
Hm, maybe it would be good to add a case of `offset = 2, repeat=undefined` to show that it results in `2, 2, 2, 2, 2, ...`